rxandroid/src/test/java/io/reactivex/android/schedulers/HandlerSchedulerTest.java

```diff
@@ -746,18 +742,6 @@ public void workerSchedulePeriodicallyInputValidation() {
             assertEquals("run == null", e.getMessage());
         }
         try {
-            worker.schedulePeriodically(new CountingRunnable(), -1, 1, MINUTES);
-            fail();
-        } catch (IllegalArgumentException e) {
-            assertEquals("delay < 0: -1", e.getMessage());
-        }
-        try {
-            worker.schedulePeriodically(new CountingRunnable(), 1, -1, MINUTES);
-            fail();
-        } catch (IllegalArgumentException e) {
-            assertEquals("period < 0: -1", e.getMessage());
```

Comment:
@JakeWharton This has been amended, I don't know if you expect a ping, but in any case here it is.